src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubInvocationMessage.cs

```diff
@@ -5,12 +5,25 @@
 
 namespace Microsoft.AspNetCore.SignalR.Protocol
 {
+    /// <summary>
+    /// A base class for hub invocation messages.
```

Comment:
hub ~~invocation~~ messages **which relate to a specific invocation**

Since it's also for things like Completion, etc.